packages/language-server/src/completion/completions.ts

```diff
@@ -369,6 +369,8 @@ export function getSuggestionForBlockTypes(lines: string[]): CompletionList {
 
   const isViewAvailable = Boolean(previewFeatures?.includes('views'))
 
+  const isTypeAvailable = Boolean(datasourceProvider?.includes('mongodb'))
```

Comment:
Unrelated to this PR, but this sort of "feature query" that gives different answers depending on the DB connector really should be exposed as a Wasm method you could just call from TS.
This stuff is going to break silently as soon as we add a new DB connector.

Reply:
Ideally, by the time we add a new connector, all of this stuff will be in engines 🤞  :x but I get your point